src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -157,7 +157,9 @@ private function joinRelations(QueryBuilder $queryBuilder, QueryNameGeneratorInt
             }
 
             if (false === $propertyMetadata->isReadableLink() || false === $propertyMetadata->isReadable()) {
-                continue;
+                if (false === $propertyMetadata->getAttribute('fetchEager', false)) {
```

Comment:
not really :|. I tried things in my head but it doesn't seem to end up well because it's not a condition to break but rather a condition to not break when we should have.